src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -411,11 +411,34 @@ internal virtual NamedTypeSymbol GetNativeIntegerType(NamedTypeSymbol underlying
             throw ExceptionUtilities.Unreachable();
         }
 
+        public bool SupportsRuntimeCapability(RuntimeCapability capability)
+        {
+            // Keep in sync with VB's AssemblySymbol.SupportsRuntimeCapability
+            switch (capability)
+            {
+                case RuntimeCapability.ByRefFields:
+                    return this.RuntimeSupportsByRefFields;
+                case RuntimeCapability.CovariantReturnsOfClasses:
+                    return this.RuntimeSupportsCovariantReturnsOfClasses;
+                case RuntimeCapability.DefaultImplementationsOfInterfaces:
+                    return this.RuntimeSupportsDefaultInterfaceImplementation;
+                case RuntimeCapability.NumericIntPtr:
+                    return this.RuntimeSupportsNumericIntPtr;
+                case RuntimeCapability.UnmanagedSignatureCallingConvention:
+                    return this.RuntimeSupportsUnmanagedSignatureCallingConvention;
+                case RuntimeCapability.VirtualStaticsInInterfaces:
+                    return this.RuntimeSupportsStaticAbstractMembersInInterfaces;
+            }
+
+            return false;
```

Comment:
>return false; [](http://example.com/codeflow?start=12&length=13)

It is probably worth documenting that ```false``` is returned for an unrecognized input (rather than throw).

Reply:
will do.